src/js/common/utils/helpers.js

```diff
@@ -210,7 +210,13 @@ export function dateDiffDesc(date, userFromDate = null) {
     return formatDiff(minuteDiff, 'minute');
   }
 
-  return 'less than a minute';
+  const secondDiff = date.diff(fromDate, 'seconds');
```

Comment:
This function isn't used by anything at the moment so changing it doesn't break anything.

Reply:
is this `dateDiffDescription` or `dateDiffDescending`?

Reply:
I changed the name in the code, hopefully it's more clear now.